_opensearch/install/rpm.md

```diff
@@ -5,154 +5,472 @@ parent: Install OpenSearch
-   ```
-
-   ```bash
+   # Install the x64 package using rpm.
    sudo rpm -ivh opensearch-{{site.opensearch_version}}-linux-x64.rpm
-   sudo rpm -ivh opensearch-dashboards-{{site.opensearch_version}}-linux-x64.rpm
    ```
-  
    **arm64**
-
    ```bash
+   # Install the arm64 package using yum.
    sudo yum install opensearch-{{site.opensearch_version}}-linux-x64.rpm
-   sudo yum install opensearch-dashboards-{{site.opensearch_version}}-linux-arm64.rpm
-   ```
-
-   ```bash
+   # Install the arm64 package using rpm.
    sudo rpm -ivh opensearch-{{site.opensearch_version}}-linux-x64.rpm
-   sudo rpm -ivh opensearch-dashboards-{{site.opensearch_version}}-linux-arm64.rpm
    ```
-  
-   Once complete, you can run OpenSearch inside your distribution.
-
-## YUM method
-
-YUM, an RPM package management tool, allows you to pull the RPM package from the YUM repository library. 
-
-1. Create a repository file for both OpenSearch and OpenSearch Dashboards:
-
+1. After the installation succeeds, enable OpenSearch as a service.
+    ```bash
+    sudo systemctl enable opensearch
+    ```
+1. Start OpenSearch.
+    ```bash
+    sudo systemctl start opensearch
+    ```
+1. Verify that OpenSearch launched correctly.
+    ```bash
+    sudo systemctl status opensearch
+    ```
+
+### Install OpenSearch from a local YUM repository
+
+YUM, the primary package management tool for Red Hat-based operating systems, allows you to download and install the RPM package from the YUM repository. 
+
+1. Create a local repository file for OpenSearch:
    ```bash
-   sudo curl -SL https://artifacts.opensearch.org/releases/bundle/opensearch/2.x/opensearch-2.x.repo -o /etc/yum.repos.d/opensearch-2.x.repo
+   sudo curl -SL https://artifacts.opensearch.org/releases/bundle/opensearch/{{major_version_mask}}/opensearch-{{major_version_mask}}.repo -o /etc/yum.repos.d/opensearch-{{major_version_mask}}.repo
    ```
-
-   ```bash
-   sudo curl -SL https://artifacts.opensearch.org/releases/bundle/opensearch-dashboards/2.x/opensearch-dashboards-2.x.repo -o /etc/yum.repos.d/opensearch-dashboards-2.x.repo
-   ```
-  
-   To verify that the repos appear in your repo list, use `sudo yum repolist`.
-
-2. Clean your YUM cache, to ensure a smooth installation:
-
+1. Verify that the repository was created successfully.
+    ```bash
+    sudo yum repolist
+    ```
+1. Clean your YUM cache, to ensure a smooth installation:
    ```bash
    sudo yum clean all
    ```
-
-3. With the repository file downloaded, list all available versions of OpenSearch and OpenSearch-Dashboards:
-
+1. With the repository file downloaded, list all available versions of OpenSearch:
    ```bash
    sudo yum list opensearch --showduplicates
-   sudo yum list opensearch-dashboards --showduplicates
    ```
-
-4. Choose the version of OpenSearch you want to install: 
-
+1. Choose the version of OpenSearch you want to install: 
+   - Unless otherwise indicated, the latest available version of OpenSearch is installed.
```

Comment:
I wanted to include the modifier here because latest version available, to me as a reader, acknowledges that sometimes the "latest version" of OpenSearch may not yet be available (due to any number of issues such both within our control and outside it)